client/i18n/locales/english/intro.json

```
@@ -366,7 +366,7 @@
       "In the Front End Development Libraries Certification, you'll learn how to style your site quickly with Bootstrap. You'll also learn how to add logic to your CSS styles and extend them with Sass.",
       "Later, you'll build a shopping cart and other applications to learn how to create powerful Single Page Applications (SPAs) with React and Redux."
     ],
-    "note": "",
+    "note": "Note: React 18 may be incompatible with some front-end challenges.",
```

```suggestion
    "note": "Note: React 18 may be incompatible with the certification projects.",
```

I believe it's the certification projects (with the external test suite) that are the issue.